### PR TITLE
[DOCUMENTATION] - Update broken links and format of `WINDOWS_11.md`

### PR DESCRIPTION
**Pull request by brandondorner**
There were some discrepancies with broken links and broken formatting in the `WINDOWS_11.md` file relative to the [sharepoint document](https://boozallen.sharepoint.com/teams/VABID/appeals/_layouts/15/Doc.aspx?sourcedoc={20830a1e-4aae-459b-80d9-64b76d8dfebf}&action=edit&wd=target%28Developer%20Setup.one%7C319614ad-d7af-4973-aeb4-a8418a647fb8%2FWindows%2011%20%2B%20WSL%20Setup%7Cfcab41ab-dfc1-45c7-b763-c4ba42fbbc61%2F%29&wdorigin=NavigationUrl)

This commit addresses those issues.

- Add in missing links
- Fix broken links
- Add in code snippets formatting
- Update formatting of nested lists

## Acceptance Criteria
- [ ] Code compiles correctly
- [ ] The added links match correctly with the [sharepoint doc](https://boozallen.sharepoint.com/teams/VABID/appeals/_layouts/15/Doc.aspx?sourcedoc={20830a1e-4aae-459b-80d9-64b76d8dfebf}&action=edit&wd=target%28Developer%20Setup.one%7C319614ad-d7af-4973-aeb4-a8418a647fb8%2FWindows%2011%20%2B%20WSL%20Setup%7Cfcab41ab-dfc1-45c7-b763-c4ba42fbbc61%2F%29&wdorigin=NavigationUrl)

 BEFORE
image
 ---|---
AFTER
image

### CODE REVIEW

File: WINDOWS_11.md [hunk 1]
```diff
@@ -82,45 +83,48 @@ Run each line is a separate command, run them one at a time
...
 
 19. Run source scripts/ubuntu_setup.sh
-    a. This built out Facols 2nd time and removes a intermediate container.
-    b. Run the source scripts/ubuntu_setup.sh a third time. This should do a final check for missing dependancies.
+    * a. This built out Facols 2nd time and removes a intermediate container.
+    * b. Run the source `scripts/ubuntu_setup.sh` a third time. This should do a final check for missing dependancies.
 
 20. (In directory: cd ~/appeals/caseflow): In terminal: perform run commands:
-    a. make run-backend
+    * a. make run-backend
```
Comment (LiannaNewman) [inline WINDOWS_11.md:90]: To keep the same syntax as other parts of the document i'd recommend changing make run-backend to ```make run-backend```

**Review by LiannaNewman — CHANGES_REQUESTED** [review]
Looks good! Great job updating the number steps as I know the numbering is a bit off in the OneNote document. Just recommending one quick change and this should be good to go.

File: WINDOWS_11.md (hunk 1 above)
Comment (brandondorner) [reply WINDOWS_11.md]: Good catch!

**Review by LiannaNewman — APPROVED** [review]
This is ready to go!

**Pull request merged**